p/scripts/main.js

```diff
@@ -1087,8 +1087,19 @@ function init_stream(stream) {
 		}
 
 		el = ev.target.closest('.item.share > button[data-type="clipboard"]');
-		if (el && navigator.clipboard) {	// Clipboard
-			navigator.clipboard.writeText(el.dataset.url);
+		if (el) { // Clipboard
+			if (navigator.clipboard) {
+				navigator.clipboard.writeText(el.dataset.url);
```

Comment:
With reference to https://github.com/FreshRSS/FreshRSS/pull/5606/files#r1302688688
```suggestion
				navigator.clipboard.writeText(el.dataset.url)
					.catch(e => alert(e));
```

Reply:
PS As far as I'm concerned none of this stuff should be part of this PR, mind you. Just some potential future thoughts should you wish to look into the matter further.

I also noticed there doesn't really seem to be any feedback, so ideally it'd look more like this.

```suggestion
				navigator.clipboard.writeText(el.dataset.url)
					.then(() => {
						// UI feedback
					})
					.catch(e => alert(e)); // needn't be an alert really, could also use a similar mechanism as the positive feedback above
```

Reply:
Thanks. I took it into the code. Please have a look